src/components/editor/InputLiteral.jsx

```diff
@@ -76,9 +77,10 @@ export class InputLiteral extends Component {
     this.setState({ content_add: usr_input })
   }
 
-  notRepeatable(userInputArray,currentcontent){
-    if (this.props.formData == undefined){
+  notRepeatable(userInputArray, currentcontent){
+    if (this.props.formData == undefined || this.props.formData.items < 1){
       this.addUserInput(userInputArray, currentcontent)
+      this.setState({ disabled: true})
```

Comment:
Is the whitespace between the opening brace and `disabled:` intentional or stray?

Reply:
we're not terribly consistent on that particular whitespace, and we've been ignoring the inconsistency.

Reply:
Definitely a stray